iree_tests/.gitattributes

```diff
@@ -0,0 +1,2 @@
+*.irpa filter=lfs diff=lfs merge=lfs -text
+*.mlirbc filter=lfs diff=lfs merge=lfs -text
```

Comment:
Looks like GitHub puts a file size limit on LFS files of 5GB. That's pretty small for real ML model weights.
https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage

GitHub also charges per download... I wonder what our limits are (free tier is 1GB/month... lol)
https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-storage-and-bandwidth-usage